test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/ServiceUnavailablePageViewSpec.scala

```diff
@@ -38,7 +40,7 @@ class ServiceUnavailablePageViewSpec extends UnitViewSpec {
     }
 
     "have sign out link" in {
```

Comment:
Perhaps capture the reason for this sign out link being "different" in the test name, maybe something like

```suggestion
    "have sign out link (for users that cannot access the NIDAC service)" in {
```